index.html

```diff
@@ -45,7 +45,7 @@
 				<li><a class="scroll-link" data-target="schedule" href="#schedule">Schedule</a></li>
 				<li><a class="scroll-link" data-target="location" href="#location">Venue</a></li>
 				<li><a class="scroll-link" data-target="faq" href="#faq">FAQ</a></li>
-				<li><a class="scroll-link" data-target="partners" href="#partners">Partners</a></li>
+				<li><a class="scroll-link" data-target="partners" href="#sponsors">Partners</a></li>
```

Comment:
I didn't feel like duplicating/refactoring all the CSS which is dependent on the `sponsors` ID...